### PR TITLE
Allow to extend additional info key, value to OpenAPI Info section

### PR DESCRIPTION
**Pull request by chenatlas**
Hey @vitalik! Thanks for your previous feedback and I totally get not being locked into an engine.
I wonder what do you think about this solution?
adding `additional_info` as optional to initialize the API, so users can pass whatever they need to extend the OpenAPI 'info' section.
 
for example:
```Python
api = NinjaAPI(
               version="1.0.0",
               additional_info={
                   "termsOfService": "https://example.com/terms/",
                   'x-logo': {'url': 'https://example.com/logo.svg'}
               },
               title="Demo API",
               description="This is a demo API with dynamic OpenAPI info section")
```

### CODE REVIEW

**Comment by SmileyChris** [comment]
A nice simple addition. Bikeshedding the name, potentially this should be `openapi_info` to indicate that it's openAPI related.

Looks good, but could do with a test and docs. 👍🏻

**Review by baseplate-admin — CHANGES_REQUESTED** [review]
LGTM, Please add test and documentation

**Comment by chenatlas** [comment]
@baseplate-admin / @SmileyChris done both, added testing, renamed to openapi_info and added docs.

File: docs/docs/guides/api-docs.md [hunk 1]
```diff
@@ -51,12 +65,11 @@ from django.urls import reverse
 
 reverse('api-1.0.0:openapi-view')
 
->>> '/api/docs'
```
Comment (SmileyChris) [inline docs/docs/guides/api-docs.md:54]: I think this should stay, it's showing the output of the reverse

**Review by SmileyChris — CHANGES_REQUESTED** [review]

**Comment by chenatlas** [comment]
@SmileyChris fixed, moved openapi_info to last and reverted the emitted line.

**Comment by vitalik** [comment]
Hi @chenatlas thank you for contribution, this is merged, but I extended functionality to support other extra open api attributes (not just `info`)

this is how updating info should work now:
```PY
api = NinjaAPI(
   openapi_extra={
            "info": {
                "termsOfService": "https://example.com/terms/",
            }
        }
       
   },
```

**Comment by joaoflaviosantos** [comment]
Hello @vitalik!

Is it possible to include extra schemas (not related to a specific route)? For example, as below?

```python
api = NinjaAPI(
    openapi_extra={
        "components": {
            "schemas": {
                "TestSchema": {
                    "title": "TestSchema",
                    "type": "object",
                    "properties": {
                        "cod_test": {"title": "Cód. test", "type": "integer"},
                        "test": {
                            "title": "Test",
                            "maxLength": 30,
                            "type": "string",
                        },
                    },
                    "required": ["cod_test", "test"],
                },
            }
        }
    }
)
```

Or using Schemas/model Schemas directly:

```python
api = NinjaAPI(
    openapi_extra={
        "components": {
            "schemas": {
                "TestSchema": TestSchema().dict(),
            }
        }
    }
)
```

Thanks in advance for the help!

**Pull request merged**